ui/packages/catalog-ui-search/src/main/webapp/component/editable-rows/editable-rows.view.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
❓ I don't necessarily disagree with the compositional style you're going for here, but I think what you're aiming for with this file would typically be achieved by a more comprehensive `.hbs` template that is then used in a single view.

Reply:
Basically I'd like to get some other input from @codice/ui as to whether or not this is a direction we'd like to move in the future (or if anyone has concerns with this that I'm not seeing)

Reply:
Is this comment OBE now after our discussion?